### PR TITLE
changed first file

### PR DESCRIPTION
**Pull request by NayuduRama**
## Hightlights
* I just changed the file to see how git works.

### CODE REVIEW

File: general/First.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class First {
     public static void main(String[] args) {
-        System.out.println("Helo world!");
+        System.out.println("Helo Australia!");
```
Comment (ChinnaBabuSadam) [inline general/First.java]: please change it to Inida.

File: general/First.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class First {
     public static void main(String[] args) {
```
Comment (ChinnaBabuSadam) [inline general/First.java]: no args,  use different name

Comment (NayuduRama) [reply general/First.java]: this is convention
please let us use this.

File: general/First.java (hunk 1 above)
Comment (NayuduRama) [reply general/First.java]: changed.

**Pull request merged**